package.json

```diff
@@ -19,32 +19,34 @@
   "author": "",
   "license": "ISC",
   "peerDependencies": {
+    "next": "^12",
     "react": ">=17",
     "react-dom": ">=17",
-    "next": "^12"
+    "raw-body": ">=2.5.0"
```

Comment:
Why peer?

Reply:
This package is required only in case of using nextjs helpers (createSaleorEventHandler use nextjs request types)